tests/run.bats

```diff
@@ -36,19 +36,19 @@ load helpers
 
 	# This should succeed, because buildah run stops caring at the --, which is preserved as part of the command.
 	run_buildah --debug=false run $cid echo -- -n test
-	is "$output" "-- -n test" "echo"
+	expect_output -- "-- -n test"
 
 	# This should succeed, because buildah run stops caring at the --, which is not part of the command.
 	run_buildah --debug=false run $cid -- echo -n -- test
-	is "$output" "-- test" "echo"
+	expect_output -- "-- test"
 
 	# This should succeed, because buildah run stops caring at the --.
 	run_buildah --debug=false run $cid -- echo -- -n test --
-	is "$output" "-- -n test --" "echo"
+	expect_output -- "-- -n test --"
```

Comment:
Perfectly fine. The intention here is to say "stop processing arguments"; otherwise, `expect_output()` sees the `-- -n test --` and tries to parse `--` as a flag and the comparison fails.